include/mxnet/engine.h

```diff
@@ -33,24 +33,7 @@ typedef Opr* OprHandle;
  * \brief OnComplete Callback to the engine,
  *  called by AsyncFn when action completes
  */
-class CallbackOnComplete {
```

Comment:
It was intended to not use std::function on this case, to save additional space and virtual function cost